travis-run-tests.sh

```diff
@@ -1,22 +1,24 @@
 #! /usr/bin/env bash
 
-set -x
-
 current_version=`python setup.py --version`
-if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" != false ]]
+
+if ! git diff-index --quiet master openfisca*/**
```

Comment:
Also be careful with the `'!'`, I'm not sure it means `== 0` in Bash, or else it is a construct I don't know.